zuulcilint/utils.py

```diff
@@ -124,6 +124,9 @@ def get_zuul_object_from_yaml(
     except yaml.YAMLError:
         print(f"Invalid YAML in Zuul YAML file: {zuul_yaml_file}", file=sys.stderr)
         sys.exit(1)
+    except AttributeError:
```

Comment:
When is this exception raised?

Reply:
When trying to lint paths that don't contain zuul config files .. this way we exit with a clean error

Reply:
When trying to lint paths that don't contain zuul config files .. this way we exit with a clean error

Reply:
Why that exception specifically?

Reply:
> Checking playbook paths
> Traceback (most recent call last):
>   File "/home/pbaptista/Workspace/private/zuulcilint/venv/bin/zuulcilint", line 6, in <module>
>     sys.exit(main())
>   File "/home/pbaptista/Workspace/private/zuulcilint/zuulcilint/__main__.py", line 322, in main
>     invalid_playbook_paths = lint_playbook_paths(zuul_good_yaml)
>   File "/home/pbaptista/Workspace/private/zuulcilint/zuulcilint/__main__.py", line 109, in lint_playbook_paths
>     jobs = zuul_utils.get_zuul_object_from_yaml(ZuulObject.JOB, file_path)
>   File "/home/pbaptista/Workspace/private/zuulcilint/zuulcilint/utils.py", line 120, in get_zuul_object_from_yaml
>     return [obj for obj in yaml.safe_load(f) if obj.get(obj_type.value)]
>   File "/home/pbaptista/Workspace/private/zuulcilint/zuulcilint/utils.py", line 120, in <listcomp>
>     return [obj for obj in yaml.safe_load(f) if obj.get(obj_type.value)]
> AttributeError: 'str' object has no attribute 'get'